src/main/java/com/luka/gamesellerrating/config/SecurityConfig.java

```diff
@@ -78,7 +81,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity httpSecurity) throws
     @Bean
     public CorsConfigurationSource corsConfigurationSource() {
         CorsConfiguration configuration = new CorsConfiguration();
-        configuration.setAllowedOrigins(List.of(baseUrl, keycloakProperties.getAuthServerUrl()));
+        configuration.setAllowedOrigins(List.of("http://locahost:" + port, keycloakProperties.getAuthServerUrl()));
```

Comment:
What about app in btp? Will it work if only localhost is configured? Moreover I see a TYPO http://locahost: -> should be localhost

+ If you want to still use it, please store it as constant